lib/widgets/post/widgets/post-body/widgets/post_body_text.dart

```diff
@@ -21,12 +21,31 @@ class OBPostBodyText extends StatelessWidget {
         onLongPress: _copyText,
         child: Padding(
             padding: EdgeInsets.all(20.0),
-            child: OBActionableSmartText(
-              text: _post.getText(),
-            ))
+            child: _buildActionablePostText()
+      )
     );
   }
 
+  Widget _buildActionablePostText() {
+    return StreamBuilder(
+        stream: this._post.updateSubject,
+        initialData: this._post,
+        builder: (BuildContext context, AsyncSnapshot<Post> snapshot) {
+          Post post = snapshot.data;
+
+          if (post.isEdited != null && post.isEdited) {
```

Comment:
Interestingly, this `snapshot.data` does not return the updated text after a post is edited successfully, I have checked the `updatableModel` , `updateFromJson` method which fires and sets the new text correctly. But when the `notifyUpdate` runs and the stream builder fires, the snapshot still has the old data. I tried adding keys to the `ObPostBodyText` widgets etc too but no luck.
Thoughts?